cmd/util/flags/spec.go

```diff
@@ -73,11 +72,11 @@ func SpecFlags(settings *SpecFlagSettings) *pflag.FlagSet {
 		settings.EnvVar,
 		`The environment variables to supply to the job (e.g. --env FOO=bar --env BAR=baz)`,
 	)
-	flags.Float64Var(
+	flags.Int64Var(
 		&settings.Timeout,
 		"timeout",
 		settings.Timeout,
-		`Job execution timeout in seconds (e.g. 300 for 5 minutes and 0.1 for 100ms)`,
+		`Job execution timeout in seconds (e.g. 300 for 5 minutes)`,
```

Comment:
Might this flag, like `max-timeout` be nicer as a duration?

Reply:
From what I can see specifying no unit is (rightly) an error for duration values. Today we accept values with no unit as seconds, so this would be backwards-incompatible. However it should be pretty obvious and definitely a better UX, so I'd be happy to change this if we think the impact is low?

Reply:
I'm not sure I've a good grip on what level of breakage we're comfortable with atm, so perhaps safer to leave it like this.